contracts/evoting/evoting.go

```diff
@@ -821,6 +834,108 @@ func (e evotingCommand) manageAdminForm(snap store.Snapshot, step execution.Step
 	return nil
 }
 
+// TODO modify
```

Comment:
is this TODO comment still needed? if not, please remove it, if so, please add some details so that some future developer knows what to do